mlrun/__main__.py

```diff
@@ -1450,5 +1461,49 @@ def func_url_to_runtime(func_url, ensure_project: bool = False):
     return runtime
 
 
+def load_notification(notifications: str, project: mlrun.projects.MlrunProject):
+    """
+
```

Comment:
either remove the line or add title to what the method does, title might be redundant as it is pretty straight forward